packages/mgt-components/src/components/mgt-profile/mgt-profile.ts

```diff
@@ -493,8 +495,14 @@ export class MgtProfile extends BasePersonCardSection {
   }
 
   private getDisplayDateRange(event: EducationalActivity): string {
+    // if startMonthYear is not defined, we do not show the date range (otherwise it will always start with 1970)
+    if (!event.startMonthYear) {
+      return null;
```

Comment:
let's use the `nothing` value that lit provides instead of null here please

Reply:
will do